contact_calc/hydrophobics.py

```diff
@@ -98,6 +98,10 @@ def compute_hydrophobics(traj_frag_molid, frame_idx, index_to_atom, sele1, sele2
         if atom1.chain == atom2.chain and abs(atom1.resid - atom2.resid) < res_diff:
             continue
 
+        #Check and continue if disulphide bond
+        if atom1.resid in disulfide_cys and atom2.resid in disulfide_cys:
```

Comment:
I think this would discard the H-bond between backbone N and O atoms of two different cysteines which should not happen.